src/WebJobs.Extensions.DocumentDB/Services/DocumentDBService.cs

```diff
@@ -66,6 +67,19 @@ public IOrderedQueryable<DocumentCollection> CreateDocumentCollectionQuery(Uri c
             return response.Resource;
         }
 
+        public async Task<DocumentQueryResponse> ExecuteNextAsync(Uri documentCollectionUri, string sqlExpression, string continuation)
+        {
+            FeedOptions feedOptions = new FeedOptions { RequestContinuation = continuation, EnableCrossPartitionQuery = true };
+            IDocumentQuery<Document> query = _client.CreateDocumentQuery<Document>(documentCollectionUri, sqlExpression, feedOptions).AsDocumentQuery();
+            FeedResponse<Document> response = await query.ExecuteNextAsync<Document>();
```

Comment:
I get the below error when I run this in a 32-bit process (which xunit is by default). I've started a conversation with the DocDB team:

`Partition routing information cannot be extracted from the query when running in a 32-bit process. To complete your query and avoid this exception, ensure that your host process is 64-bit. For Executable projects, this can be done by unchecking the "Prefer 32-bit" option in the project properties window, on the Build tab.`